js/compiler/compiler.go

```diff
@@ -194,16 +243,57 @@ func newBabel() (*babel, error) {
 	return result, err
 }
 
-// Transform the given code into ES5, while synchronizing to ensure only a single
+func increaseMappingsByOne(sourceMap []byte) ([]byte, error) {
```

Comment:
minor nitpick, but this function probably deserves a comment for _why_ it exists and does what it does